src/kubernetes_api_objects/persistent_volume_claim.rs

```diff
@@ -78,7 +78,7 @@ impl PersistentVolumeClaim {
     #[verifier(external_body)]
     pub fn api_resource() -> (res: ApiResource)
         ensures
-            res@.kind == Kind::CustomResourceKind,
+            res@.kind == Kind::PersistentVolumeClaimKind,
```

Comment:
Thanks for catching the copy-paste bug. I think we should just do `res@.kind == PersistentVolumeClaimView::kind()` (`kind()` should not take self, my bad).

Let's do it in the next PR.